Sitka-Sound-Science-Center/sitka-landslide
# Add data quality warning banner & change NWS API grid point

## Overview

This PR converts the old, existing beta banner to a new data quality warning banner, including renaming of the relevant component and CSS module. EDIT: It also changes the NWS API grid point, rather than making a separate PR.

Resolves #66 & #65

### Demo

![Screen Shot 2022-09-29 at 3 16 54 PM](https://user-images.githubusercontent.com/77936689/193122570-d7c651fa-6fe5-4206-857e-914cc16ff7c6.png)


### Notes

The existing, eye-catching style of the beta banner seemed to work for this purpose, so I've just reused it as is; however, the one thing I'm not sold on is the blue text of the "National Weather Service" link on that blue background. I played with alternative colors and styles such as underlining and bolding, but didn't find the perfect way to signal that it was a link without that characteristic blue text. See a few attempts below:

![Screen Shot 2022-09-29 at 3 21 45 PM](https://user-images.githubusercontent.com/77936689/193123640-2971bbd0-c2ac-4d75-b820-2bc791f26ae0.png)

![Screen Shot 2022-09-29 at 3 22 30 PM](https://user-images.githubusercontent.com/77936689/193123652-51ec9a2d-0faa-411f-b8c1-11356646c868.png)

Ultimately, I've left it as is because the blues seem different enough to be legible (even with color blind filters applied) but am open to trying something else.


## Testing Instructions

 * Start the server and go to any page
 * Verify the banner appears at the top


## Checklist
